---
float32 power  # The current rollerbrush power level
